phpcs.xml.dist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="loop">
+  <description>PHP CodeSniffer configuration for ereolen base</description>
+
+  <file>sites/all/modules</file>
+  <file>sites/all/themes</file>
+
+  <exclude-pattern>sites/all/default</exclude-pattern>
+  <exclude-pattern>includes/</exclude-pattern>
+  <exclude-pattern>misc/</exclude-pattern>
+  <exclude-pattern>modules/</exclude-pattern>
+  <exclude-pattern>profiles/</exclude-pattern>
+  <exclude-pattern>scripts/</exclude-pattern>
+  <exclude-pattern>themes</exclude-pattern>
+  <exclude-pattern>vendor/</exclude-pattern>
+  <exclude-pattern>*\.features\.*</exclude-pattern>
+  <exclude-pattern>*\.(field_group|strongarm|pages_default|views_default|rules_defaults|path_breadcrumbs)\.inc</exclude-pattern>
+  <exclude-pattern>README.md</exclude-pattern>
+
+  <!-- Show progress -->
+  <arg value="p"/>
+
```

Comment:
`phpcs` should be able to detect the Drupal version, but to be sure that the right version is used, you can add

```
  <config name="drupal_core_version" value="7"/>
```